File: solutions/javascript/rna-transcription/1/rna-transcription.js
const DNA_TO_RNA = {
  'G': 'C',
  'C': 'G',
  'T': 'A',
  'A': 'U'
};

export const toRna = (dna) => {
  return dna.replace(/[GCTA]/g, (nucleotide) => DNA_TO_RNA[nucleotide]);
};

File: solutions/swift/rna-transcription/1/Sources/RnaTranscription/RnaTranscription.swift
func toRna(_ dna: String) -> String {
    let complements: [Character: Character] = [
        "G": "C",
        "C": "G",
        "T": "A",
        "A": "U"
    ]
    
    return String(dna.compactMap { complements[$0] })
}
